yaml/common.yaml

```diff
@@ -162,6 +162,30 @@ sensor:
             ESP_LOGI("SET", "Updating humidity to %f", x);
             id(gFEUCHTE) = x;
 
+#########################################
+#                                       #
+#   Template Sensors                    #
+#                                       #
+#########################################
+  - platform: template
+    name: "COP WW TAG"
+    update_interval: $interval_very_slow
+    state_class: "measurement"
+    lambda: |-
+      if (id(WAERMEERTRAG_WW_TAG_SUMME_KWH).state && id(EL_AUFNAHMELEISTUNG_WW_SUMME_KWH).state)
```

Comment:
add a check for non-zero demoninators: 

if (id(WAERMEERTRAG_WW_TAG_SUMME_KWH).state && id(EL_AUFNAHMELEISTUNG_WW_SUMME_KWH).state && id(EL_AUFNAHMELEISTUNG_WW_SUMME_KWH).state != 0)